### PR TITLE
Simplify single page example.

### PR DESCRIPTION
**Pull request by stonebk**
Seems like we can remove some of the optional stuff for the simple example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,29 +34,25 @@
 ```jsx
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { createStore, combineReducers, applyMiddleware } from 'redux';
+import { createStore } from 'redux';
 import { createInputsReducer, connectWithInputs, ReduxInputsWrapper } from 'redux-inputs';
 import { Provider } from 'react-redux';
-import thunk = from 'redux-thunk';
```
Comment (stonebk) [inline README.md]: Is thunk absolutely required?

File: README.md [hunk 2]
```diff
@@ -34,29 +34,25 @@
...
 
 const inputsConfig = {
     email: {
         defaultValue: 'test@example.com',
         validator: (value) => typeof value === 'string' && value.indexOf('@') >= 0
     }
 };
-const reducer = combineReducers({
```
Comment (stonebk) [inline README.md]: We only have one reducer in this example, don't really need to combine.

File: README.md [hunk 3]
```diff
@@ -34,31 +34,27 @@
...
+const EmailInput = ReduxInputsWrapper(({id, value, error, onChange}) => (
     <div>
         <input name={id} value={value} onChange={(e) => onChange(e.target.value)}/>
         {error ? <p style={{color: 'red'}}>Your email must contain an @</p> : null}
     </div>
-);
-EmailInput = ReduxInputsWrapper(EmailInput);
+));
 
-function Form(props) {
```
Comment (stonebk) [inline README.md]: For consistency.

File: README.md (hunk 2 above)
Comment (stonebk) [inline README.md]: I guess we do need to specify because redux inputs creates its own state at another mount point.

File: README.md (hunk 1 above)
Comment (stonebk) [inline README.md]: Shoot, and it appears the thunk is required. Can we not require this unless we are using async validators??

File: README.md (hunk 2 above)
Comment (stonebk) [inline README.md]: Hmm, doesn't look like this can be removed?

File: README.md (hunk 1 above)
Comment (jakepusateri) [inline README.md]: redux-thunk is just a requirement of using the library. Having to go and change configuration to use some of the features is oddly inconsistent.

**Comment by jakepusateri** [comment]
I like the two-step EmailInput/WrappedEmailInput. Reassigning is confusing.

**Pull request closed without merging**